db/controllers/articles.controller.js

```diff
@@ -21,3 +21,21 @@ exports.getArticleById = (req, res, next) => {
     .catch(next);
 }
 
+exports.updateArticleVotes = (req, res, next) => {
+    const articleId = req.params.article_id;
+    const { inc_votes } = req.body;
+
+    if (!Number.isInteger(inc_votes)) {
+        return res.status(400).send({ msg: "number of votes must be a valid integer" });
+    }
```

Comment:
Should be a promise reject rather than directly invoking res.
I'd also expect to see promise rejects handeled in the model rather than the controller.

A question too the answer might be no, does PSQL throw an error if we try and input anything other than a number?